packages/origin-backend/src/pods/organization/organization-invitation.service.ts

```diff
@@ -88,20 +88,21 @@ export class OrganizationInvitationService {
 
         const invitation = await this.invitationRepository.findOne(invitationId, {
             where: {
-                email: user.email,
-                status: Not(
-                    In([
-                        OrganizationInvitationStatus.Accepted,
-                        OrganizationInvitationStatus.Rejected
-                    ])
-                )
```

Comment:
What is wrong with this query?

Reply:
There is nothing wrong with it per se, but for some reason this it crashes on some installations:

```
2020-08-17T07:42:38.880632+00:00 app[web.1]: [Nest] 3   - 08/17/2020, 7:42:38 AM   [ExceptionsHandler] invalid input syntax for type integer: "{"_type":"not","_value":{"_type":"in","_value":[2,1],"_useParameter":true,"_multip2020-08-12020-08-17T07:42:38.880632+00:00 app[web.1]: [Nest] 3   - 08/17/2020, 7:42:38 AM   [ExceptionsHandler] invalid input syntax for type integer: "{"_type":"not","_value":{"_type":"in","_value":[2,1],"_useParameter":true,"_multip2020-08-17T07:42:38.880632+00:00 app[web.1]: [Nest] 3   - 08/17/2020, 7:42:38 AM   [ExceptionsHandler] invalid input syntax for type integer: "{"_type":"not","_value":{"_type":"in","_value":[2,1],"_useParameter":true,"_multipleParameters":true},"_usePa2:38.880632+00:00 app[web.1]: [Nest] 3   - 08/17/2020, 7:42:38 AM   [ExceptionsHandler] invalid input syntax for type integer: "{"_type":"not","_value":{"_type":"in","_value":[2,1],"_useParameter":true,"_multipleParameters":true},"_useParameter":true,"_multipleParameters":false}" +1ms
```

so decided to rollback and replace with code check. Query worked fine in e2e tests and our canary deployments.